### ISSUE TITLE
Extract more data from wikidata

### ISSUE THREAD

**Issue by dalf** [searx/searx#560]
More data could be extracted from wikidata and be shown in the infoboxes.

For example searching for ["searx"](http://searx.me/?q=!wikidata+searx&category_general=on) the license and the programming language could be shown (see the [wikidata page](https://www.wikidata.org/wiki/Q17639196))

Technical note : with the used API, the result contains only Q1131681 and Q28865 without the names (for two mentioned fields). 

But using the `parse` API the problem can be solved. Example for the searx page : 

https://www.wikidata.org//w/api.php?action=parse&format=json&uselang=en&page=Q17639196&redirects=1&prop=text%7Clanglinks%7Crevid%7Cwikitext&disableeditsection=1&disabletidy=1&preview=1&sectionpreview=1&disabletoc=1&utf8=1&formatversion=2

the HTML in the "text" entity can be parsed easily since there are `id` and `class` attributes

**Comment by MarcAbonce** [comment]
currently working on this here: https://github.com/a01200356/searx/commits/wikidata

**Comment by dalf** [comment]
Thanks a lot !
I've created a wiki page about the properties that can extracted : https://github.com/asciimoo/searx/wiki/wikidata

**Comment by MarcAbonce** [comment]
I added many, but not all, of the properties in the list. I tried to give as much relevant information without overwhelming the user with too much data (for example printing the full cast of a film would be too much, imo).
Nevertheless, it should be easy to add, remove or change the order of properties in the engine, so this feature can be tweaked as it gets used.

The extra requests are still synchronous though, so I just kept result_count to 1 to avoid problems.
